src/GitLinkTask/GitLink.targets

```diff
@@ -1,16 +1,18 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
-  <UsingTask TaskName="GitLink" AssemblyFile="$(MSBuildThisFileDirectory)GitLinkTask.dll" />
+  <UsingTask TaskName="LinkPdbToGitRemote" AssemblyFile="$(MSBuildThisFileDirectory)GitLinkTask.dll" />
 
-  <Target Name="LinkSymbolsToRemoteGitRepo" AfterTargets="Compile">
-    <GitLink Condition=" Exists('$(IntermediateOutputPath)$(TargetName).pdb') "
+  <Target Name="LinkSymbolsToRemoteGitRepo"
```

Comment:
Wouldn't it make more sense to name this "LinkSymbolsToRemote" to stay consistent with the task name?